File: examples/forex/quote/node/getAFxQuote.js
const { ForexService } = require("m3o/forex");

// Get the latest quote for the forex
async function getAfxQuote() {
  let forexService = new ForexService(process.env.MICRO_API_TOKEN);
  let rsp = await forexService.quote({
    symbol: "GBPUSD",
  });
  console.log(rsp);
}

getAfxQuote();

File: examples/go/forex/quote/getAFxQuote.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/forex"
)

// Get the latest quote for the forex
func GetAfxQuote() {
	forexService := forex.NewForexService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := forexService.Quote(&forex.QuoteRequest{
		Symbol: "GBPUSD",
	})
	fmt.Println(rsp, err)
}
